client/htlc/cli/sendtx.go

```diff
@@ -18,7 +22,7 @@ func GetCmdCreateHTLC(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create an HTLC",
-		Example: "iriscli htlc create --receiver=<receiver> --receiver-on-other-chain=<receiver-on-other-chain> --amount=<amount> --hash-lock=<hash-lock> " +
+		Example: "iriscli htlc create --to=<to> --receiver-on-other-chain=<receiver-on-other-chain> --amount=<amount> --secret=<secret> " +
```

Comment:
Seems `--from` is missing :)

Reply:
Looks should add `--from`, `--chain-id`, `--fee`.